test/irb/test_command.rb

```diff
@@ -210,6 +210,98 @@ def test_irb_info_lang
     end
   end
 
+  class CustomCommandTestCase < CommandTestCase
+    def setup
+      super
+      execute_lines("show_cmds\n") # To ensure command initialization is done
```

Comment:
nit: let's use `help` here:

```suggestion
      execute_lines("help\n") # To ensure command initialization is done
```